Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.py

```diff
@@ -322,6 +378,24 @@ def risk_detection_get_command(client: Client, args: Dict[str, Any]) -> CommandR
                           raw_response=raw_response)
 
 
+def test_module(client: Client):
+    """Tests API connectivity and authentication'
+    The test module is not functional, it raises the suitable exceptions according to the selected authentication type.
+
+    Args:
+        client (Client): Azure Risky Users API client.
+
+    Returns: None
+    """
+    if client.authentication_type == CLIENT_CREDENTIALS_FLOW:  # Client credentials flow
```

Comment:
are we unable to run azure-risky-users-auth-test here?

Reply:
I copied the exact logic from the 'Microsoft-365-defender' integration that supports client credentials flow.
 I tend to agree with you that It would be better to call it here instead of asking the user to do so. 
@yucohen - Is there a reason for not calling the 'azure-risky-users-auth-test' here?

Reply:
@JudahSchwartz We can't use the test-module here since we are getting the access token from the integration context, which is unavailable from the test button. Also, as customers already used to using the auth-test command in those integrations, I think it's better leaving it as is.

Reply:
During the initial setup isnt the only case a user might hit the test connection button. After a user fully configures the integration they should be able to click test and get a green response. 
See Yuval's implementation here https://github.com/demisto/content/pull/20441/files

Reply:
I improved the test module so that in client credentials auth mode the connection test occurs without asking the user to run commands.